main.go

```diff
@@ -1,13 +1,17 @@
 package main
 
 import (
-	_ "github.com/mdg-iitr/Codephile/conf"
+	"os"
+
 	"github.com/astaxie/beego"
-	_ "github.com/mdg-iitr/Codephile/routers"
 	sentryhttp "github.com/getsentry/sentry-go/http"
+	_ "github.com/mdg-iitr/Codephile/conf"
+	_ "github.com/mdg-iitr/Codephile/routers"
 )
 
 func main() {
+	beego.BConfig.RunMode = os.Getenv("ENVIRONMENT")
+
```

Comment:
Is this required now?

Reply:
@Saksham-Sharma-99

Reply:
i'm removing it ..sorry got busy in some other work.

Reply:
done :)